driver/CMakeLists.txt

```diff
@@ -107,4 +108,11 @@ if(ENABLE_DKMS)
 
 endif()
 
+if(INSTALL_PROBE_LOADER)
+	install(PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/../scripts/sysdig-probe-loader
+		DESTINATION bin
+		RENAME ${PROBE_NAME}-loader
+		COMPONENT agent-slim)
```

Comment:
What is agent-slim? I don't see any reference in the project.

Reply:
Just like agent-kmodule elsewhere, it's a(n unfortunate) reference to the Sysdig Monitor (commercial) agent. Either this, or we duplicate code between OSS and the commercial code base. Or we come up with a better idea, thoughts?

Reply:
Can't the commercial agent just apply a patch to change this? Falco does the same in many places and it just feels right.

Reply:
Technically, I guess so. But let's just agree to disagree whether that feels right :)

Reply:
TBH, I personally don't believe that's a good idea to reference a non-OSS component inside an OSS project.
Moreover, that confuse users who don't have access to the whole codebase. 

Just my 2 cents

Reply:
I totally agree and I don't like it either. It's just we have three bad solutions to pick from. I could also make the component name a variable and see how cmake reacts.